setup.py

```diff
@@ -7,6 +7,11 @@
 
 from setuptools import setup, find_packages
 
+# Make the current nilearn sources importable.
```

Comment:
@GaelVaroquaux I did my best to avoid the `import nilearn` and figure out how to call functions, will push the changes I made shortly.